wordpress/containerbuddy/onstart.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Refresh all config files in order
+#
+# This script is typically called once at the container start, but
+# it can be called manually if WP config details must be changed
+
+# The database and memcached config files are separate to avoid collisions
+# if their backends' onChange handlers are triggered simultaneously
+echo "******running onstart script*********"
+
+until [[ `curl -s consul:8500/v1/health/state/passing | grep mysql-primary` ]]
```

Comment:
It took me some time to find this: `$CONSUL_HOST/v1/health/service/$SERVICE_NAME?passing`, but perhaps you'd rather use it here?